stories/setup/Provider.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+// Without core/polyfill we see the following error:
+// "regeneratorRuntime is not defined".
+import 'core/polyfill';
+import createStore from 'amo/store';
+import { addQueryParamsToHistory } from 'core/utils';
+
+const history = addQueryParamsToHistory({
+  history: createBrowserHistory(),
+});
+
+const { store } = createStore({ history });
+
+export default function Provider({ story }: Object) {
```

Comment:
I don't see this used anywhere. Maybe you need it for a forthcoming patch? If that's the case, I'd rather see this all applied to the forthcoming patch.